imapautofiler/actions.py

```diff
@@ -299,6 +299,119 @@ def invoke(self, conn, mailbox_name, message_id, message):
         )
 
 
+class Flag(Action):
+    r"""Set, add or remove one or more message flags.
+
+    The action is indicated with the name ``flag``.
+
+    The action expects either a "set" or an "add" and / or a "remove" entry,
+    which specify the flag(s) to set or add and / or remove. The entry value
+    can either be a single flag or a list of flags. IMAP defines several
+    *system flags*, which start with a backslash, e.g. ``\Seen``. You can use
+    one of the following constants to refer to these system flags::
+
+        ANSWERED = r'\Answered'
+        DELETED = r'\Deleted'
+        DRAFT = r'\Draft'
+        FLAGGED = r'\Flagged'
+        SEEN = r'\Seen'
```

Comment:
"Seen" is the IMAP standard, but I'm not sure most users think of it that way. Most clients use "read" and "unread". Maybe we could make them aliases?

Reply:
Since they are the inverse of each other, you need to add `\Seen` to mark a message as read, and remove `\Seen` to mark it as unread. A 1:1 alias would therefor not work with the current configuration scheme. And turning `add: UNREAD' into `remove: SEEN` seems prone to cause confusion, esp. in conjunction with other flags in the same action.